asm/banks/c3.asm

```diff
@@ -2640,7 +2687,102 @@ string_bet:
 .case_empty
   LDA #$FF                ; space character
   JSR string_fill         ; fill item name with spaces
-  RTS    
+  RTS
+
+; ------------------------------------------------------------------------
+; New Helpers for condensed colosseum menu
+
+load_item_desc:
+  JSR $8308          ; set desc ptrs
+  TDC                ; zero A/B
+  LDA $4B            ; cursor slot
+  LSR                ; A: row index, C: column index (0/1)
+  TAX                ; index row
+  LDA !colo_items,X  ; item in slot
+  BCC .load          ; branch if left-side column
+  JSR GetPrize       ; get prize for this wager
+.load
+  JMP $5738          ; Load description
+
+; ------------------------------------------------------------------------
+; Loop go on seeking until we find an item that returns a reward and
+; then store both the ID and Prize in buffers
+
+GetPrize:
+  REP #$20          ; 16-bit A
+  ASL #2            ; wager item x4
+  TAX               ; index into colosseum data
+  SEP #$20          ; 8-bit A
+  TDC               ; zero B
+  LDA $DFB602,X     ; prize item ID
+  RTS
+
+Set_Arrow:
+  LDA #$75          ; load new colosseum limit
+  STA $5C           ; set ^
+  LDA #$FF          ; max scrollbar speed
+  STA $7E357A       ; set ^
```

Comment:
This can probably be shorted to `STA $357A`

Reply:
`STA $357A` don't work.